ingestion/templates/secrets.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name}}-geocoding-secrets
+type: Opaque
+stringData:
+  auth-id: {{ .Values.smartyAuthId}}
```

Comment:
With the type of secret being `stringData`, you'll need to change these slightly:
`{{ .Values.smartyAuthId | quote }}`

Otherwise, the parser throws an exception when you try to run `helm install`. 

`* Secret in version "v1" cannot be handled as a Secret: json: cannot unmarshal number into Go struct field Secret.stringData of type string`

Reply:
Huh... I didn't get that error when I did the install nor the dry install, I'll take a look

Reply:
Changed